packages/core/js-sdk/src/admin/api-key.ts

```diff
@@ -0,0 +1,104 @@
+import { FindParams, HttpTypes, PaginatedResponse } from "@medusajs/types"
+import { Client } from "../client"
+import { ClientHeaders } from "../types"
+
+export class ApiKey {
+  private client: Client
+  constructor(client: Client) {
+    this.client = client
+  }
+
+  async list(queryParams?: FindParams, headers?: ClientHeaders) {
+    return await this.client.fetch<
+      PaginatedResponse<HttpTypes.AdminApiKeyListResponse>
+    >(`/admin/api-keys`, {
+      query: queryParams,
+      headers,
+    })
+  }
+
+  async create(body: HttpTypes.AdminCreateApiKey, headers?: ClientHeaders) {
```

Comment:
For other create/update endpoints we also allow passing params, to filter whats returned